### ISSUE TITLE
Некорректно считает пополнение в $

### ISSUE THREAD

**Issue by Atorich** [alexey-goloburdin/tinkoff-analytics#1]
Привет)
Затестил на своем портфеле и обнаружил, что пополнение в $ считается как рублевое.

```
eugene@mi-notebook:~/tinkoff-analytics$ export TINKOFF_ACCOUNT_STARTED=29.11.2020
eugene@mi-notebook:~/tinkoff-analytics$ python3 go.py
Текущий курс доллара в брокере: 73.3475 руб
Пополнения: 1 089 руб
```

Хотя 29 ноября было пополнение на $1089, а не 1089 руб. 

Отсюда общая сумма пополнений ниже, а прибыль - выше реальной, т.к. текущая стоимость в рублях учитывает доллары по курсу )

**Comment by alexey-goloburdin** [comment]
Да, я писал для ИИС, его только рублями пополнять можно, баксы уже внутри покупаются. Допишу для пополнения баксами

**Comment by Atorich** [comment]
Это было пополнение долларового счета с рублевого Black, просто автоконвертация по рынку.
Т.е. не пополнение рублевого, а потом покупка валюты с брокерского, а сразу покупка баксов.
(вдруг это поможет)
